linkstack/docker-compose.yml

```
@@ -0,0 +1,49 @@
+version: "3.8"
+
+services:
+
+  linkstack:

```

Adding a comment indicating that this image doesn't support a custom user
```suggestion
  linkstack:
    # This image doesn't support a custom user
    # user: '1000:1000'
```

---

I have now implemented all suggested changes. I hope everything is correct so far.